lib/llhttp/parser.js

```diff
@@ -87,16 +80,12 @@ const wasm_on_header_value = (p, at, length) => {
 
 const wasm_on_headers_complete = p => {
   currentParser[kHeaderSize] = 0
-  const versionMajor = inst.exports.llhttp_get_http_major(p)
-  const versionMinor = inst.exports.llhttp_get_http_minor(p)
   const rawHeaders = currentParser[kGetHeaders]()
   const statusCode = inst.exports.llhttp_get_status_code(p)
-  const statusMessage = currentParser[kStatusMessage]
```

Comment:
If we are never going to need this, I am in favor. Just asking to be sure.

Reply:
I don’t think ao